bark/commons/transformation/frenet_state.hpp

```diff
@@ -22,22 +22,30 @@ struct FrenetState : public FrenetPosition {
   FrenetState() : FrenetPosition() {}
   FrenetState(const double& longitudinal, const double& lateral,
               const double& vlongitudinal, const double& vlateral,
-              const double& angle)
+              const double& angle, double angleRoad, double vLatRoad)
       : FrenetPosition(longitudinal, lateral),
         vlon(vlongitudinal),
         vlat(vlateral),
-        angle(angle) {}
+        angle(angle),
+        angleRoad(angleRoad) {}
   FrenetState(const bark::models::dynamic::State& state,
               const bark::geometry::Line& path);
 
   double vlon;
```

Comment:
I think we should change those namings or at least add a documentation to it. vlat is misleading, I would prefer to use d_lat_dot or so